libraries/ESP8266WebServer/src/detail/mimetable.cpp

```diff
@@ -5,48 +5,93 @@
 namespace mime
 {
 
-// Table of extension->MIME strings stored in PROGMEM, needs to be global due to GCC section typing rules
-const Entry mimeTable[maxType] PROGMEM = 
+static const char kHtmlSuffix[] PROGMEM = ".html";
+static const char kHtmSuffix[] PROGMEM = ".htm";
+static const char kCssSuffix[] PROGMEM = ".css";
```

Comment:
Missing MIMETYPE_MINIMAL conditional

Reply:
The conditional is not strictly needed, the linker will remove the strings that are not referenced. I can add an explicit conditional.

Reply:
Conditional added.